docs/adr/adr_zxing_replacement.md

```diff
@@ -0,0 +1,46 @@
+# ADR: Replacing ZXing library for QR Code reading
+
+Trello-card: [Link to Task](https://trello.com/c/Zv00dw2j/1864-20-fe-replace-zxing-qr-reading-package?filter=member%3Amomorazor)
+
+Decision Deadline: N/A
+
+Author: Maurovic Cachia
+
+## Status
+
+Proposed
+
+## Context or Problem Statement
+
+The current library used by the frontend to parse QR Codes has been in maintenance mode for the past 2 years. It would be ideal to move the QR Code reading to library that is still actively worked on, so as not to run into issues as devices continue to change and older software gets more and more out of date.
+
+## Decision Drivers
+
+Maintaining QR Code reader functionality, increasing longevity and fixing some rare issues that already occure on certain devices.
```

Comment:
I would group these points under the headings:
1. Reliability: (What you said about functionality and longevity)
2. Maintainabilty + Bug Fixing: (what you said about issues increasing as devices change and older software gets more out of date)